main.go

```diff
@@ -0,0 +1,129 @@
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/7uu13/forum/models"
+	_ "github.com/mattn/go-sqlite3"
+)
+
+func main() {
+
+	db, err := sql.Open("sqlite3", "mydb.db")
+	if err != nil {
+		panic(err)
+	}
+
+	defer db.Close()
+
+	models.PerformMigrations(db)
+
```

Comment:
In the main functions, we should only have database migrations, nothing else, so please get rid of the other functions, like insert user and etc.